app/views/E2EResetYourPasswordView.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useLayoutEffect } from 'react';
+import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+
+import { E2EEnterYourPasswordStackParamList } from '../stacks/types';
+import { useTheme } from '../theme';
+import { e2eResetOwnKey } from '../lib/services/restApi';
+import { showConfirmationAlert, showErrorAlert } from '../lib/methods/helpers';
+import { logout } from '../actions/login';
+import log, { events, logEvent } from '../lib/methods/helpers/log';
+import Button from '../containers/Button';
+import * as HeaderButton from '../containers/Header/components/HeaderButton';
+import KeyboardView from '../containers/KeyboardView';
+import SafeAreaView from '../containers/SafeAreaView';
+import I18n from '../i18n';
+import sharedStyles from './Styles';
+import scrollPersistTaps from '../lib/methods/helpers/scrollPersistTaps';
+
+const styles = StyleSheet.create({
+	info: {
+		fontSize: 16,
+		lineHeight: 24,
+		marginTop: 24,
+		...sharedStyles.textRegular
+	},
+	content: {
+		gap: 32
+	}
+});
+
+interface IE2EResetYourPasswordView {
+	navigation: NativeStackNavigationProp<E2EEnterYourPasswordStackParamList, 'E2EResetYourPasswordView'>;
+}
+
+const E2EResetYourPasswordView = ({ navigation }: IE2EResetYourPasswordView): React.ReactElement => {
```

Comment:
Are we intentionally duplicating this flow?
If you didn't discuss that with design, I'd say this is wrong.

I think they actually wanted to redirect to E2EEEncryptionSecurityView. Not create a new screen.
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/1977c48213735524e10f400951a517d64136efe0/app/views/E2EEncryptionSecurityView/index.tsx#L77